netbox_cmdb/netbox_cmdb/models/interface.py

```diff
@@ -0,0 +1,42 @@
+from django.core.validators import MaxValueValidator, MinValueValidator
+from django.db import models
+from netbox.models import ChangeLoggedModel
+
+FEC_CHOICES = [
+    (None, "None"),
+    ("rs", "Reed Solomon"),
+    ("fc", "FireCode"),
+]
+
+
+class DeviceInterface(ChangeLoggedModel):
+    """A device interface configuration."""
+
+    name = models.CharField(max_length=100)
+    device = models.ForeignKey(
+        to="dcim.Device",
+        on_delete=models.CASCADE,
+        related_name="%(class)sdevice",
+        null=False,
+        blank=False,
+    )
+    autonegotiation = models.BooleanField(default=True)
+    speed = models.PositiveIntegerField(
+        validators=[MinValueValidator(1), MaxValueValidator(800000000)],
```

Comment:
just curious why this value?

Reply:
I used the same units and kind of constraints as https://github.com/netbox-community/netbox/blob/9b325f4b8611160f62294cfdb6bafebae2757bdd/netbox/dcim/choices.py#L1139

but we can remove it (if you're talking about MinValue/Maxvalue)